package.json

```diff
@@ -20,6 +22,8 @@
   "devDependencies": {
     "@types/marked": "^0.6.0",
     "@types/puppeteer-core": "^1.9.0",
+    "http-server": "^0.11.1",
```

Comment:
I don't think I want to force developers to install `http-server`. Perhaps they would prefer Apache because they already have it installed or another alternative like `serve`.

Reply:
Works for me.  The main motivation behind this PR was to simplify the contributing process, but it already seems pretty simple so perhaps I can combine the existing commands into one without having to re-wire everything.

Reply:
The reason why they are separate is because changes to the backend require you to restart node. Maybe you can use `npx nodemon` instead of `node` so it will restart on change.